docs/x509/reference.rst

```diff
@@ -2542,6 +2543,17 @@ instances. The following common OIDs are available as constants.
 
         Corresponds to the dotted string ``"2.5.29.24"``.
 
+Helper Functions
+~~~~~~~~~~~~~~~~
+.. currentmodule:: cryptography.x509
+
+.. function:: random_serial_number()
+
+    .. versionadded:: 1.6
+
+    Generates a random serial number suitable for use when constructing
+    certificates or CRLs.
```

Comment:
There actually is a `CRLNumber` extension that is used as a serial number for CRLs, but entropy in it isn't a requirement. I'll remove the language.